src/app/tests/suites/certification/Test_TC_WNCV_1_1.yaml

```diff
@@ -42,8 +42,7 @@ tests:
       response:
           value: 3
 
-    - label: "read the optional global attribute: FeatureMap"
-      disabled: 1
+    - label: "read the global attribute: FeatureMap"
       command: "readAttribute"
       attribute: "FeatureMap"
       response:
```

Comment:
Why is the expected value here 0?  I'd think it would be 1 based on the XML, but the .zap file for all-clusters-app seems to override this to "0".  window-app.zap has "0x0001" for the defaultValue, as I would expect...

Reply:
You are right, this is because  there is a bug (understood now) on the endpoint config. Will waiting i wanted to set 0 but i hope Vivien can solve it quickly.  I've notified  @vivien-apple about it

What is happening is  ZAP_LONG_DEFAULTS_INDEX(6343) },             /* FeatureMap */  
Does point over the same byte field organization  for LittleEndian or BigEndian ->  0x00, 0x00, 0x00, 0x01,
so 1 is currently read 16777216 -> error

I am gonna hold until Vivien or someone else fix this
@bzbarsky-apple should i fill an issues on this ?

Reply:
@jmeg-sfy If we're outputting incorrect default values in big endian, then yes, please!

Reply:
@jmeg-sfy @tecimovic @vivien-apple This is a bug in ZAP, in fact.  In `src-electron/generator/helper-endpointconfig.js` in the `endpoint_attribute_long_defaults` function there is no endianness handling apart from the comments that get spit out.  Compare what happens in `endpoint_attribute_list`...

I filed https://github.com/project-chip/zap/issues/218

Reply:
We gonna hold until this is merged
https://github.com/project-chip/connectedhomeip/pull/9725

Reply:
@jmeg-sfy @bzbarsky-apple - did we get the zap fix in? This  PR starts being stale ... should there be some followup?